client/src/pages/Login/Login.tsx

```diff
@@ -39,18 +41,31 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={12} md={12} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
+          <AppBar position="sticky" className={classes.header}>
+            <LogoHeader logo={logo} />
+            <AuthHeader linkTo="/login" asideText="" btnText="LOGIN" />
+            <AuthHeader linkTo="/signup" asideText="" btnText="SIGNUP" />
+          </AppBar>
```

Comment:
We shouldn't be defining multiple navbars in various components, instead we should make it a part of the layout. Some parts of the navbar will naturally be different based on authentication but this is where we can use a single conditional rendering to display the appropriate subcomponent.